src/Uno.UI/UI/Xaml/XamlRoot.crossruntime.cs

```diff
@@ -15,6 +15,12 @@ public sealed partial class XamlRoot
 
 	internal event Action? RenderInvalidated;
 
+#if __SKIA__
+	// For profiling purposes only. Do not depend on these events.
+	internal event Action? PaintedFrame;
```

Comment:
For consistency I would adjust these to `FramePainted` and `FrameRendered` (similarly to `RenderInvalidated` etc)